docs/sql-guide/functions/function-identifier.md

```diff
@@ -0,0 +1,48 @@
+---
+title: IDENTIFIER()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# IDENTIFIER() function
+
+`IDENTIFIER()` function generates unique IDs for the provided index.
+
+## Syntax
+
+```
+identifier(expr)
```

Comment:
If there's plans to expand the `expr` to something more (e.g., a select within the brackets), then that identifier is fine.

If it's only ever going to be a table name, then just call it that:

```sql
identifer(table_name)
```
I say this because it's clearer at first glance.

Reply:
Based on the example, it looks like a value is also required (or optional). I recently did work on the SET functions to clarify this for users.

So for example, we'd change the syntax to something like:

```sql
IDENTIFIER('table-name'[,<values>])
```
This means the Arguments will look something like this:

## Arguments

| Argument | Description | Data type | Additional information |
|---|---|---|---|
| `table_name` | Existing table to add row identifer | `string` | [String data type](/docs/sql-guide/data-types/data-type-string) |
| `<values>` | Optional value that conforms to the destination column data type | Valid data types | [FeatureBase Data types](/docs/sql-guide/data-types/data-types-home) |